src/Modules/FudgeTools.psm1

```diff
@@ -1757,6 +1762,8 @@ function Invoke-Chocolatey
     # set the parameters to pass
     $ParametersArg = Format-ChocolateyParams $Parameters
 
+    $Arguments = ($Arguments + ' --allow-unofficial')
```

Comment:
Should be able to replace this with `$Arguments += ' --allow-unoffical'` 😛 

Also, do we want to set this flag on every platform, or just Unix?

Reply:
I don't see why people shouldnt be able to build and use their own version of choco on Windows also.

Anyone using Fudge is very unlikely to be *unexpectedly* getting a hacked version of choco; and if they did ... the hacker would surely remove the code which checks for officialness.

Reply:
Fair enough 👍